src/features/staking/StakingModal.tsx

```diff
@@ -2,7 +2,6 @@ import { FC, RefObject, useState } from "react";
 import { useForm } from "react-hook-form";
 import { ImSpinner8 } from "react-icons/im";
 import { VscChromeClose } from "react-icons/vsc";
-import { useForm } from "react-hook-form";
```

Comment:
If these are small changes that don't warrant a branch that's totally fine, just a note to check that it isn't an error.

Reply:
Actually This was a change that was made in the "small bug fixes" branch. I made the same change on this branch because it was breaking the app. This change won't happen again in this PR